_benchmark/user-guide/optimizing-benchmarks/custom-runners.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Custom runners
+nav_order: 130
+parent: Optimizing benchmarks
+grand_parent: User guide
+---
+
+# Adding custom runners
+
+**Runners** are operation types performed on an OpenSearch cluster. You can add customer runners for operations that don't already exist in a workload.
```

Comment:
> for operation types that do not exist in OpenSearch Benchmark (OSB).
Could we also link the types of operations that exist in OSB from the documentation?